ocs_ci/ocs/resources/test_packagemanifest.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf8 -*-
+from unittest.mock import patch
+
+import pytest
+
+from ocs_ci.ocs.resources.packagemanifest import PackageManifest
+from ocs_ci.ocs.exceptions import (
+    ResourceNameNotSpecifiedException,
+    ResourceNotFoundError,
+)
+
+
+def test_pm_null():
+    """
+    Test that creation of PackageManifest object without any constructor
+    agruments works (object is created, no exceptions are raised).
+    """
+    PackageManifest()
+
+
+def test_pm_null_get_default_channel():
```

Comment:
Do we have a test description?

Reply:
Is it necessary?

Reply:
Not required but the others have them so I figured it would be a nice to have.